src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -27,7 +31,10 @@ public DataProcessor(List<RoadAccident> roadAccidentList){
      * @return
      */
     public RoadAccident getAccidentByIndex7(String index){
-        return null;
+        Optional<RoadAccident> foundRoadAccident = roadAccidentList.stream()
+                .filter(accident -> accident.getAccidentId().equals(index))
+                .findFirst();
+        return foundRoadAccident.isPresent()?foundRoadAccident.get():null;
```

Comment:
That's one way of getting values from Optional. Another one is to use https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html#orElse-T- 

optional.orElse(null);